src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/entities/DraftAdjudication.kt

```diff
@@ -21,6 +22,9 @@ data class DraftAdjudication(
   @OneToOne(optional = true, cascade = [CascadeType.ALL], fetch = FetchType.EAGER)
   @JoinColumn(name = "incident_role_id")
   var incidentRole: IncidentRole,
+  @OneToMany(cascade = [CascadeType.ALL], orphanRemoval = true, fetch = FetchType.EAGER)
+  @JoinColumn(name = "draft_adjudication_fk_id")
+  var offenceDetails: MutableList<Offence>? = null,
```

Comment:
Not sure that its any better but I guess you could have:
var offenceDetails: MutableList<Offence> = mutableListOf<>()
Then the list wouldn't ever be nullable.

Reply:
Null is useful - it shows it hasn't been added yet